### PR TITLE
Connect to psql instance

### PR DESCRIPTION
**Pull request by JedraPeake**
- updated readme where applicable
- setup connection to psql instance, once your local environment variables are updated
- added user table based on https://dbdiagram.io/d/5e8648fb4495b02c3b892c24
- added base user routes, getall, getid, update, patch, delete

### CODE REVIEW

**Comment by samjam48** [comment]
relates #3

**Comment by caiokf** [comment]
are we gonna go with graphql or api style endpoints?

**Comment by samjam48** [comment]
I can run the server and seems to login and go to the endpoint okay

I get an error when I try running `knex migrate:latest` or `node_modules/.bin/knex migrate:latest`

`server-syncify git:(psql) knex migrate:latest
Using environment: development
Error: connect ECONNREFUSED 127.0.0.1:5432
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1141:16)`

Looking on the interweb apparently might be related to docker settings. Do I need to docker compose something first?

https://stackoverflow.com/questions/53572374/error-connect-econnrefused-127-0-0-15432-docker-migrate

**Comment by caiokf** [comment]
@samjam48 you would need to have postgres running localyy on that port. through docker or otherwise

**Comment by JedraPeake** [comment]
> are we gonna go with graphql or api style endpoints?

@caiokf  I honestly don't know what the decision is on this. I was more focused on getting psql connection where we can keep the schema in sync with a migration folder between developers. I'm happy to update it as required or remove it for the pr.

@samjam48 currently docker isn't set up, so you need to have psql running locally. I'm happy to take a closer look if that's not your issue

File: README.md [hunk 1]
```diff
@@ -26,52 +26,61 @@ This is a NodeJS project with Express, GraphQL, PSQL and Auth0 login
...
-use etherpay
+The application is currently using [bookshelf.js](https://bookshelfjs.org/) and [knex.js](http://knexjs.org/)
 
 ---
 
-### Installation
+### GraphQL
 
-`npm install`
+Testing GraphQL querys/mutations can be done through [GraphIQL](https://graphql.org/learn/queries/)
```
Comment (caiokf) [inline README.md:41]: shouldn't this be a link to our `localhost:xxxx/graphiql`?

File: README.md [hunk 2]
```diff
@@ -26,52 +26,61 @@ This is a NodeJS project with Express, GraphQL, PSQL and Auth0 login
...
+DB_HOST=127.0.0.1
 ```
 
 ---
 
+### Installation
+
+`npm install`
+
+`knex migrate:latest` or `node_modules/.bin/knex migrate:latest`
```
Comment (caiokf) [inline README.md:80]: i would create a target to able able o run `npm run migrate` instead

**Comment by caiokf** [comment]
I didn't try to run it locally, but code looks good. Well done @JedraPeake. I left some minor comments but from what I see, we should merge this.

File: README.md (hunk 2 above)
Comment (JedraPeake) [reply README.md:80]: I did this earlier today on the docker branch, I will update it

File: README.md (hunk 1 above)
Comment (JedraPeake) [reply README.md:41]: I was pointing it at the documentation so poeple could see how to build the query. I'll update it so I have both and its a little more clear

**Pull request merged**